---
layout: page
title: Software
permalink: /projects/
---

Visit me on [Github](http://github.com/lukfor)

## Cloudgene

Cloudgene is a freely available platform to improve the usability of MapReduce programs by providing a graphical user interface for the execution, the import and export of data and the reproducibility of workflows on in-house (private clouds) and rented clusters (public clouds).

The aim of Cloudgene is to build a standardized graphical execution environment for currently available and future MapReduce programs, which can all be integrated by using its plug-in interface.

[http://cloudgene.uibk.ac.at](http://cloudgene.uibk.ac.at)


## Imputation Server

This server provides a free genotype imputation service. You can upload GWAS genotypes and receive imputed genomes in return. The underlying imputation engine is base on minimac, which implements a low memory, computationally efficient algorithm for genotype imputation that can handle very large reference panels with thousands of haplotypes. The current version of this server uses the 1000 Genomes Project phase III release as a reference for imputation.

[https://imputationserver.sph.umich.edu](https://imputationserver.sph.umich.edu)

## mtDNA Server

mtDNA-Server provides a scalable service for the analysis of mtDNA NGS data currently focusing on heteroplasmy and contamination detection. 

[http://mtdna-server.uibk.ac.at](http://mtdna-server.uibk.ac.at)

## Askimed

Askimed is a software product to collect clinical study or register data using electronic case report forms (eCRF). It is the backbone of several large-scale population studies (e.g. [GCKD](http://www.gckd.de/), [ncRNAPain](http://www.ncrna-pain.eu/)).

[http://www.askimed.com](http://www.askimed.com)


## Cloudflow

Cloudflow is a MapReduce pipeline framework, which is based on a similar concept as JavaFlume or Apache Crunch. In contrast to these existing approaches, Cloudflow was developed to simplify the pipeline creation in biomedical research, especially in the field of Genetics. For that purpose Cloudflow supports a variety of NGS data formats and contains a rich collection of built-in operations for analyzing such kind of datasets (e.g. quality checks, mapping reads or variation calling).

[https://github.com/genepi/cloudflow](https://github.com/genepi/cloudflow)


## Conan

CONAN is a freely available client-server software solution which provides an intuitive graphical user interface for categorizing, analyzing and associating CNVs with phenotypes. Moreover, CONAN assists the evaluation process by visualizing detected associations via Manhattan plots in order to enable a rapid identification of genome-wide significant CNV regions. Various file formats including the information on CNVs in population samples are supported as input data.

[http://genepi-conan.i-med.ac.at/](http://genepi-conan.i-med.ac.at/)